src/applications/disability-benefits/686c-674-v2/config/form.js

```diff
@@ -593,26 +593,26 @@ export const formConfig = {
             schema: studentEducationBenefitsPage.schema,
             depends: formData =>
               isChapterFieldRequired(formData, TASK_KEYS.report674),
-            onNavForward: ({
-              formData,
-              pathname,
-              urlParams,
-              goPath,
-              goNextPath,
-            }) => {
-              const index = getArrayIndexFromPathName(pathname);
-              const urlParamsString = stringifyUrlParams(urlParams) || '';
+            //   onNavForward: ({
```

Comment:
I commented out some additional cases in the latest commit. Is this OK @Midge-dev ?

Reply:
Yes that's fine - we plan to rework all of this during clean up. Tyvm!